<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UIAutorisation</class>
 <widget class="QDialog" name="UIAutorisation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6" stretch="0,1">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Numéro BE :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="numBELineEdit"/>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Transporteur :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="transporteurLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Produit : </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="produitComboBox">
            <item>
             <property name="text">
              <string>CLINKER</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>GYPSE</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>DOLOMITE</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>TUF</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Lieu Chargement : </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lieuChargementLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Chauffeur :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="chauffeurLineEdit"/>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Contact Chauffeur :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="contactChauffeurLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Destination :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="destinationLineEdit"/>
          </item>
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>N° Camion : </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="numCamionLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="ajouterButton">
            <property name="text">
             <string>Ajouter</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="annulerButton">
            <property name="text">
             <string>Annuler</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="tableView"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
